Xamarin.Forms.Core/SwipeDirection.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+namespace Xamarin.Forms
+{
+	[Flags]
+	public enum SwipeDirection
+	{
+		Right = 1,
+		Left = 2,
+		Up = 4,
+		Down = 8
+	}
```

Comment:
I still have to see a Swipe that is both Left and Right... otoh, I can't think of another simple way of expressing this